platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -619,7 +619,8 @@ MGL_EXPORT
 
 /**
  Value to use for a text label. Feature properties are specified using tokens
- like {field_name}.
+ like {field_name}.  (Token replacement is only supported for literal text-field
+ values--not for data-driven property functions.)
```

Comment:
Did you add this note manually after running `make style-code`? If so, it’ll be wiped away the next time someone runs `make style-code`. Make sure to put any documentation overrides in platform/darwin/scripts/style-spec-overrides-v8.json.